lightningd/options.c

```diff
@@ -512,6 +512,27 @@ static char *opt_force_channel_secrets(const char *optarg,
 	return NULL;
 }
 
+static char *opt_force_featureset(const char *optarg,
+				  struct lightningd *ld)
+{
+	char **parts = tal_strsplit(tmpctx, optarg, "/", STR_EMPTY_OK);
+	if (tal_count(parts) != NUM_FEATURE_PLACE)
+		return "Expected 5 feature sets (init, globalinit, node_announce, channel, bolt11) separated by /";
+	for (size_t i = 0; parts[i]; i++) {
+		char **bits = tal_strsplit(tmpctx, parts[i], ",", STR_EMPTY_OK);
```

Comment:
error above says "/", why does this split by ","?

Reply:
5 sets, separated by /, each a set of 0 or more feature bitnumbers separated by ,.

I didn't document thoroughly since it's a dev-only option...